MobileCenterPush/MobileCenterPushTests/Info.plist

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
<key>CFBundleSignature</key>
        <string>????</string>

Can you add this between `CFBundleShortVersionString` and `CFBundleVersion`?
Just make all configurations consistent, I'd like to have this for now.